src/device-api/middleware/apiLogger.ts

```diff
@@ -0,0 +1,21 @@
+import * as morgan from 'morgan';
```

Comment:
This is a bit nit-picky, but how about naming the folder `middlewares` instead of `middleware` as it is a collection of middlewares instead of a single middleware functionality? Similarly, I would expect if we were ever to split the API into multiple files, we would put them in a `routes` folder as that would communicate better purpose of that subfolder.

Reply:
I shall do both

Reply:
After moving v1/v2 into a routes directory, a test in v1 API is failing. I'll just rename middlewares for now. Something is wrong in the tests that needs to be debugged, in the API testing PR..